cs251tk/student/markdownify/check_submit_date.py

```diff
@@ -0,0 +1,31 @@
+import os
+from dateutil.parser import parse
+from ...common import run, chdir
+
+
+def check_dates(spec_id, username, spec, basedir):
+    """ Port of the CheckDates program from C++
+        Finds the first submission date for an assignment
+        by comparing first commits for all files in the spec
+        and returning the earliest """
+
+    basedir = os.path.join(basedir, 'students', username, spec_id)
+    dates = []
+
+    with chdir(basedir):
+        for file in spec['files']:
+
+            # Run a git log on each file with earliest commits listed first
+            stat, res, _ = run(['git', 'log', '--reverse', '--pretty=format:%ad', '--date=iso8601',

```

Comment:
Nitpicky: can you rename this to `status`, please?

```suggestion
            status, res, _ = run(['git', 'log', '--reverse', '--pretty=format:%ad', '--date=iso8601',
```